src/garage/torch/algos/maml.py

```diff
@@ -105,25 +105,22 @@ def train_once(self, runner, all_samples, all_params):
         itr = runner.step_itr
```

Comment:
is there a reason you touched this file?

Reply:
Yes. Because I change the parameter of functions in VPG and the MAML class use the VPG functions. So I need to change the code in the MAML class too.